src/website/views/components/forms/text-input/_examples.pug

```diff
@@ -53,20 +54,56 @@ h3 Disabled
         <input type="text" class="chi-input" value="Sample Text" id="example__disabled" disabled>
       </div>
 
-h3 Invalid
-p.-text Text inputs support Chi's semantic class modifier <code>-danger</code> for indicating an invalid input.
+h3 Text Input helper message
```

Comment:
We need to add the Invalid example + paragraph back. I understand there's some overlap with the Input Helper Message example but if we don't explicitly define Invalid as a separate example and explain usage, developers will use warning or another semantic state instead.

You can leave Text Input Helper Message as you have it for now and place the Invalid example above it. As we get closer to finalizing the PR, we can figure out how to best document these and possibly merge Text Input Helper Message with the validation States examples.